code/modules/mob/living/living.dm

```diff
@@ -258,6 +258,11 @@
 
 /mob/living/proc/InCritical()
 	return (health < HEALTH_THRESHOLD_CRIT && health > HEALTH_THRESHOLD_DEAD && stat == UNCONSCIOUS)
+	
+/mob/living/proc/LowBattery()
```

Comment:
Could you rework this to be a `mob/living/silicon` proc please? It's always better to typecast things to the correct path

Reply:
Moved and tested to confirm it still works, thanks!